src/components/transactions/FlowCommons/TxModalDetails.tsx

```diff
@@ -92,13 +93,14 @@ export const DetailsNumberLineWithSub = ({
   amount,
   amountUSD,
   hideSymbolSuffix,
+  color,
 }: DetailsNumberLineWithSubProps) => {
   return (
     <Row caption={description} captionVariant="description" mb={4} align="flex-start">
       <Box sx={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-end' }}>
         <Box sx={{ display: 'flex', alignItems: 'center' }}>
           <TokenIcon symbol={symbol} sx={{ mr: 1, fontSize: '16px' }} />
-          <FormattedNumber value={amount} variant="secondary14" />
+          <FormattedNumber value={amount} variant="secondary14" color={color ? color : undefined} />
```

Comment:
No need for the ternary as it will be undefined anyways if not set

Reply:
I have a ternary obsession haha